styleguide/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ama-style-guide",
-  "version": "2.37.1",
+  "version": "2.37.2",
```

Comment:
Version is tied to sprint number. This should be 2.38.0.

Reply:
Thanks @JaeBea. This is corrected.